src/MoBi.Presentation/Tasks/Interaction/InitialConditionsTask.cs

```diff
@@ -101,6 +97,15 @@ public IMoBiCommand SetNegativeValuesAllowed(TBuildingBlock initialConditions, I
          return macroCommand;
       }
 
+      protected override bool CorrectName(TBuildingBlock buildingBlock, Module module)
+      {
+         if (buildingBlock is ExpressionProfileBuildingBlock)
```

Comment:
This needs a comment here. Why is there some special stuff done for NOT ExpressionProfle? I don't understand the code